services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -285,29 +321,41 @@ meters in flow rules at all (#data.flowType flow)"() {
                         }
                 ],
                 [
-                        flowType  : "nonCentec-nonCentec",
+                        flowType  : "Noviflow-Noviflow",
```

Comment:
The search conditions should be updated too

Reply:
fixed